### PR TITLE
Feature/automation settings bar logic

### PR DESCRIPTION
**Pull request by ofahchouch-gh**
Not only the UI works but also the logic of the buttons is calling the right functionality in the dartboard now.

### CODE REVIEW

File: www/src/components/organisms/Dartboard.vue [hunk 1]
```diff
@@ -282,6 +306,9 @@ export default {
         if(this.points.length < 3) return;
         this.$emit('update', this.points);
       }, 2000);
+    },
+    automationRunning() {
+      this.runAutomation();
```
Comment (StanBankras) [inline www/src/components/organisms/Dartboard.vue:311]: You could check if the value here is true and only then call the method. Then you can get rid of the check inside the method, with the `break`.

**Comment by StanBankras** [comment]
Other than that it looks great!

File: www/src/components/organisms/Dartboard.vue (hunk 1 above)
Comment (StanBankras) [reply www/src/components/organisms/Dartboard.vue:311]: Disregard this, the break is needed.

Comment (ofahchouch-gh) [reply www/src/components/organisms/Dartboard.vue:309]: Thanks for looking at it though.

**Pull request merged**